shell/platform/darwin/ios/framework/Source/accessibility_bridge.mm

```diff
@@ -467,17 +475,20 @@ - (nullable id)accessibilityElementAtIndex:(NSInteger)index {
 
   // Create an addtional child to act as the accessibility container for the platform view.
   if (_semanticsObject.node.IsPlatformViewNode() && index == [self accessibilityElementCount] - 1) {
+    if (_platformViewSemanticsContainer) {
```

Comment:
What if the associated platformViewID has changed since this was created?

I'd probable create and update this in `UpdateSemantics` when you process a node with platfromViewId and release it there when a node does no longer have a platformViewId.

Reply:
done!